eng/common/templates/variables/common.yml

```
@@ -17,6 +17,8 @@ variables:
   value: ""
 - name: manifestVariables
   value: ""
+- name: defaultAmd64PoolImage
```

I feel like the name should call out the platform.  This helps the readability of the various pipelines for getting an understanding if running on linux vs windows.

```suggestion
- name: defaultLinuxAmd64PoolImage
```